test/models/event_test.rb

```diff
@@ -22,3 +22,22 @@ class EventTest < ActiveSupport::TestCase
     assert_equal(false, @invalid_event.valid?)
   end
 end
+
+class EventCallbacksTestCase < ActiveSupport::TestCase
```

Comment:
If `EventCallbacks` is placed in it's own file this could similarly be split into it's own file.